realtime/src/config.rs

```diff
@@ -7,9 +7,9 @@ pub struct XSynthRealtimeConfig {
 impl Default for XSynthRealtimeConfig {
     fn default() -> Self {
         Self {
-            render_window_ms: 10.0,
+            render_window_ms: 20.0,
             channel_count: 16,
-            use_threadpool: false,
```

Comment:
Probably undo these changes, we can discuss better defaults if you think these aren't good, otherwise you can override them when running the examples manually

Reply:
Oh this was staged by accident. It was part of some temporary changes I made.
I'll revert it.